examples/Distributed-example/Lite-imagenet-singal-mul-DDP-nccl-gloo.py

```diff
@@ -0,0 +1,117 @@
+import os
```

Comment:
the `singal` in file name should be `single`?

Reply:
done